src/Microsoft.AspNet.Diagnostics/project.json

```diff
@@ -2,16 +2,16 @@
     "version": "1.0.0-*",
     "description": "ASP.NET 5 Middleware for error handling, error pages, and diagnostics information.",
     "dependencies": {
+        "Microsoft.AspNet.Diagnostics.Interfaces": "1.0.0-*",
```

Comment:
I think the build type dependency would work only with ANIs. This is now a real dependency on a real type that is really linked to, no?